Classes/Projection/Doctrine/AbstractDoctrineProjector.php

```diff
@@ -20,7 +23,7 @@
  *
  * @api
  */
-abstract class AbstractDoctrineProjector extends AbstractBaseProjector
+abstract class AbstractDoctrineProjector extends AbstractBaseProjector implements AppliedEventsAwareEventListener
```

Comment:
Mh, with this all my projectors will now use the db and there's no easy way to change that.. But if you guys all agree that this is the better way to do it, I give in of course ;)